integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/e2e/openstack/OpenStackTests.java

```diff
@@ -0,0 +1,222 @@
+
+        Object[][] data = new Object[viewResponses.size()][5];
+        for (int i = 0; i < viewResponses.size(); i++) {
+            data[i][0] = testContext;
+            data[i][1] = viewResponses.get(i).getName();
+            data[i][2] = getHdpGroups();
+            data[i][3] = getHdpScaleGroup();
+            data[i][4] = new TestCaseDescription.TestCaseDescriptionBuilder()
+                    .given("is an available cloudbreak")
+                    .when("creating cluster with prewarmed image and "
+                            + viewResponses.get(i).getName() + " clusterdefinition")
+                    .then("then the cluster creation should succeed");
+        }
+        return data;
+    }
+
+    @Test(dataProvider = "openStackHdfTestsDataProvider")
+    public void testOpenStackWithDefaulkHDFBlueprintsAndPrewarmedImagesWithoutGateWaySetup(
+            TestContext testContext,
+            String clusterDefinitionName,
+            String[] groups,
+            String scaleGroup,
+            @Description TestCaseDescription description) {
+
+        testContext
+                .given(ClusterTestDto.class)
+                .withClusterDefinitionName(clusterDefinitionName)
+                .given(StackTestDto.class)
+                .withInstanceGroupsEntity(getInstanceGroupTestDtos(testContext, groups))
+                .withCluster(ClusterTestDto.class.getSimpleName())
+                .when(stackTestClient.createV4())
+                .await(STACK_AVAILABLE)
+                .when(stackTestClient.scalePostV4()
+                        .withGroup(scaleGroup)
+                        .withDesiredCount(2))
+                .await(STACK_AVAILABLE)
+                .when(stackTestClient.scalePostV4()
+                        .withGroup(scaleGroup)
+                        .withDesiredCount(1))
+                .await(STACK_AVAILABLE)
+                .when(stackTestClient.stopV4())
+                .await(STACK_STOPPED)
+                .when(stackTestClient.startV4())
+                .await(STACK_AVAILABLE)
+                .then((tc, testDto, cc) -> stackTestClient.deleteV4().action(tc, testDto, cc))
+                .validate();
+    }
+
+    @DataProvider(name = "openStackHdfTestsDataProvider")
+    public Object[][] openStackHdfTestsDataProvider() {
+        SparklessTestContext testContext = getBean(SparklessTestContext.class);
+        minimalSetupForClusterCreation(testContext);
+
+        Object[][] data = new Object[getHdfClusterDefinition().length][5];
+        for (int i = 0; i < getHdfClusterDefinition().length; i++) {
+            data[i][0] = testContext;
+            data[i][1] = getHdfClusterDefinition()[i];
+            data[i][2] = getHdfGroups();
+            data[i][3] = getHdfScaleGroup();
+            data[i][4] = new TestCaseDescription.TestCaseDescriptionBuilder()
+                    .given("is an available cloudbreak")
+                    .when("creating cluster with prewarmed image and "
```

Comment:
`.when("a stack create request with <cluster definition> AND scale AND stop AND start requests are sent")`